src/Core/Grid/Column/Type/Common/LinkColumn.php

```diff
@@ -51,6 +51,7 @@ protected function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'sortable' => true,
                 'icon' => null,
+                'fragment' => null,
```

Comment:
I think it's more understandable than fragment, wdyt? 🤔
```suggestion
                'anchor' => null,
```

Reply:
I disagree. My understanding is that `<h1 id='#header'>Header</h1>` is an anchor and in example.com/#header `#header` is an fragment directing to an anchor.

Reply:
Mmm, I admit I don't know this term It kind of makes sense But even if it's technically true I believe `anchor` is widely used (and maybe confused) for `#elemts-in-the-url` So I would favor using the term that will make more sense to the majority n this case 🤷‍♂️

I know that in the legacy Link class the term anchor was used as well https://github.com/PrestaShop/PrestaShop/blob/a3dbe77a185bcb5e31367e2941f46a691bc70983/classes/Link.php#L155
Although legacy code is not the best reference 😅. but maybe I'm influenced by it since I'm used to this naming